pipeline_steps/delivery.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+import pytz, logging
+from base import PipelineStep
+class PipelineDelivery(PipelineStep):
+    @staticmethod
+    def process_message(message):
+        logging.warning(message.current_step + " has received " + message.name + " " + str(message.id))
+        if pytz.utc.localize(datetime.utcnow()) > message.expiration_time:
+            logging.warning(message.name + " " + str(message.id) + " has expired. It will be dropped.")
+        else:
+            logging.info(str(vars(message)))
```

Comment:
Think about how you can DRY this method.  Currently, it duplicates the code from the base class.  Can you break the method into 2 and then have this class override one of them?